lib/ipni-client.js

```diff
@@ -31,9 +34,8 @@ export async function queryTheIndex(cid, providerId) {
   }
 
   let graphsyncProvider
+  const providers = []
```

Comment:
whatever we decide to name the new "providers" return field, it should be mirrored in the function body as well